Source/WebCore/platform/graphics/avfoundation/objc/ImageDecoderAVFObjC.mm

```diff
@@ -240,19 +240,6 @@ - (void)resourceLoader:(AVAssetResourceLoader *)resourceLoader didCancelLoadingR
         return adoptRef(*new ImageDecoderAVFObjCSample(WTFMove(sampleBuffer)));
     }
 
-    CGImageRef image() const { return m_image.get(); }
-    void setImage(RetainPtr<CGImageRef>&& image)
-    {
-        m_image = WTFMove(image);
-        if (!m_image) {
-            m_hasAlpha = false;
-            return;
-        }
-
-        auto alphaInfo = CGImageGetAlphaInfo(m_image.get());
-        m_hasAlpha = alphaInfo != kCGImageAlphaNone && alphaInfo != kCGImageAlphaNoneSkipLast && alphaInfo != kCGImageAlphaNoneSkipFirst;
-    }
-
```

Comment:
This removes the only place where `m_hasAlpha` is set. Please make sure this information is retained somewhere, maybe in `rawImageFromSampleBuffer()`.

Reply:
Yes you are right. I restored the alphaInfo code back.